### ISSUE TITLE
Javascript default avatar

### ISSUE THREAD

**Issue by jdarwood007** [SimpleMachines/SMF#6976]
#### Description
If you change your avatars directory, then the javascript fallback for avatars will not work.

The code in script.js is:
```
function avatar_fallback(e) {
    var e = window.e || e;
	var default_avatar = '/avatars/default.png';
	var default_url = document.URL.substr(0,smf_scripturl.lastIndexOf('/')) + default_avatar;
```
Which basically strips the index.php off the url and slaps /avatar/default.png in.

**Comment by BrickOzp** [comment]
Seems this is a fallback if the default avatar can't be found in the defined avatar directory.

I renamed my folder to avatars2 and changed the settings accordingly and the default image was served from avatars2.
Then I renamed the folder to avatars3 but did not change the settings. In this case the fallback tried to load the avatar from /avatars/default.png

So either we keep the fallback with the standard path or we could remove it.
